cmf-cli/resources/template_feed/help/from1100/Help.Package/package.json

```diff
@@ -44,6 +44,7 @@
     "mermaid": "^9.1.3",
     "prismjs": "1.29.0",
     "rxjs": "~7.8.0",
+    "ses": "^1.4.1",
```

Comment:
this needs to be contained to the affected MES versions, we do not want to keep bumping this dependency

Reply:
Added a conditional to only do this for MES versions `>= 11.0.0` and `< 11.2.0`.

Reply:
this could be done directly in the template but it's good enough